yaml/pois.yaml

```diff
@@ -414,3 +414,45 @@ filters:
     min_zoom: 15
     output:
       kind: {col: tags->whitewater}
+  - filter: {aerialway: pylon}
```

Comment:
If we're adding a `utilities` layer, would this be more appropriate to add to that instead of `pois`?